charts/rasa-x/templates/event-service-deployment.yaml

```diff
@@ -57,10 +57,17 @@ spec:
           httpGet:
             path: "/health"
             port: "http"
+          initialDelaySeconds: {{ .Values.eventService.initialProbeDelay }}
         readinessProbe:
           httpGet:
             path: "/health"
             port: "http"
+          initialDelaySeconds: {{ .Values.eventService.initialProbeDelay }}
+        args:
```

Comment:
why do we need the `args` here?

We already have it.
https://github.com/RasaHQ/rasa-x-helm/blob/master/charts/rasa-x/templates/event-service-deployment.yaml#L44-L51

Reply:
oops, I think this was a bad merge, will remove